coap-gateway/config.yaml

```diff
@@ -18,6 +18,23 @@ apis:
       keyFile: "/secrets/private/cert.key"
       certFile: "/secrets/public/cert.crt"
       clientCertificateRequired: true
+    authorization:
+      clientID: ""
+      clientSecret: ""
+      scopes: []
+      domain: ""
```

Comment:
pls rename the `domain` to `authority` - it will be the same as is in  resource-aggregate.

Reply:
updated